pkgs/tools/typesetting/tex/texlive/bin.nix

```diff
@@ -76,6 +76,9 @@ let
         url = "https://git.archlinux.org/svntogit/packages.git/plain/trunk/texlive-poppler-0.86.patch?h=packages/texlive-bin&id=60244e41bb6f1501e8ed1fc9e6b7ba8d3f283398";
         sha256 = "0pdvhaqc3zgz7hp0x3a4qs0nh26fkvgmr6w1cjljqhp1nyiw2f1l";
       })
+
+      # Needed for ghostscript>=9.50
+      ./xdvipdfmx-fix.patch
```

Comment:
Can we fetch `https://www.tug.org/svn/texlive/trunk/Build/source/texk/dvipdfm-x/spc_dvips.c?view=patch&r1=52765&r2=52764&pathrev=52765` using fetchpatch instead?

Reply:
```
    (fetchpatch {
      url = "https://github.com/TeX-Live/texlive-source/commit/ebe96188a9699053cc134084f311b876b18bdb58.patch";
      sha256 = "1sa79q7r4vjdbxlmrdg7m1cpn725kbbjbgxy88jjpglldr04zm82";
      excludes = [ "texk/dvipdfm-x/ChangeLog" ];
    })
```

Reply:
svn might be better

Reply:
:ok_hand: thanks I didn’t know about excludes

Reply:
And re-changed to make use of svn